### PR TITLE
add dot-location rule

### LINKED ISSUE #24: dot-location
https://eslint.org/docs/rules/dot-location

### PR DESCRIPTION
**Pull request by rowbi-wan**
## Suggested rule/changes(s):
* `<dot-location>` 

## Reason for addition/amendment
Requires the dot to be located after the object rather than before the property
closes #24 

@netsells/frontend - Please review

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -89,6 +89,24 @@ let array = [
...
+```js
+const item = object
+    .property;
+```
+
+##### ✅ Example of correct code for this rule:
+
+```js
+const item = object.
+    property;
```
Comment (BeccaAnderton) [inline readme.md]: Absolutely not.

**Review by BeccaAnderton — DISMISSED** [review]

**Comment by Juddling** [comment]
Matches English language conventions of having dots at the end.

Puts the names of the field at the absolute beginning of the line so increases readability

**Comment by BeccaAnderton** [comment]
@Juddling stop this nonsense.

**Comment by samturrell** [comment]
How do i ban people

**Comment by spamoom** [comment]
I think we should consider this.

**Review by Juddling — APPROVED** [review]

**Comment by rowbi-wan** [comment]
@netsells/frontend happy now?

**Comment by samturrell** [comment]
I am happy.

**Review by samturrell — APPROVED** [review]

**Pull request merged**